addons/web/static/src/core/position_hook.js

```diff
@@ -213,6 +218,13 @@ export function reposition(reference, popper, options) {
     const { top, left } = position;
     popper.style.top = `${top}px`;
     popper.style.left = `${left}px`;
+
+    if (variantKey === "fit") {
+        const styleProperty = ["top", "bottom"].includes(directionKey) ? "width" : "height";
+        popper.style[styleProperty] = reference.getBoundingClientRect()[styleProperty] + "px";
+        popper.classList.add(`fit-${position.direction}`);
```

Comment:
Remove this line please.

In an early version of this hook we used to add a class but at some point we preferred to let the hook user handle this when we introduced the `onPositioned` callback. (Besides, you add it but never remove it)

See: https://github.com/odoo/odoo/pull/83646/commits/914ba888e249f571049f3d5a287bb908ffc0c366#diff-86a5774240db76cdc93cb570596564cc6816ef4144a7c960f10cd347b7564e44L214

That is what allowed us in the next commit to move the position_hook file one level up the file tree (because it did not have any declarative style): https://github.com/odoo/odoo/pull/83646/commits/e9395d10ec6826bd11d0543c2fbbc770746db22f

Reply:
Okay thanks. I'll see what I can do with onPositionned